processing file asmFiles/cc1.asm (1 of 3)
processing file asmFiles/cc2.asm (2 of 3)
processing file asmFiles/cc3.asm (3 of 3)
processing file asmFiles/test.coherence1.asm (1 of 2)
processing file asmFiles/test.coherence2.asm (2 of 2)
processing file asmFiles/dual.llsc.asm (1 of 2)
processing file asmFiles/dual.mergesort.asm (2 of 2)
processing file asmFiles/palgorithm.asm (1 of 1)
--- memsim.hex	2021-12-05 00:55:52.762165715 -0500
+++ memcpu.hex	2021-12-05 00:55:52.686162993 -0500
@@ -151,11 +151,9 @@
 :0400F60003E000081B
 :0400F7000000BFFC4A
 :040FFE0000000008E7
-:0418000000007A4327
-:0418010000002B14A4
-:041802000000CCB264
-:041FFB0000009975D4
-:041FFC000000CCB263
+:0418000000000A00DA
+:04180200000050058D
+:041FFB00000050058D
 :041FFD000000026C72
 :041FFE0000000208D5
 :00000001FF
